src/component/Collection/CollectionDetailItemComponent.js

```diff
@@ -57,16 +66,52 @@ export default class CollectionDetailItemComponent extends Component {
       return `선택된 트랙 (${ this.getTrackLength(trackList) })`;
     }
   }
-
-
+  
   handleSearchMapNameOnChange(e) {
     this.setState({ mapInputValue: e.target.value });
   }
 
+  handleMouseEnter(track) {
+    this.setState({ isShowPopup: true, selectedTrack: track });
+  }
+
+  handleMouseLeave() {
+    this.setState({ isShowPopup: false });
+  }
+
+  handleMouseMove(event) {
+    this.setState({
+      mouseX: event.clientX,
+      mouseY: event.clientY,
+    });
+  }
+
+  createPortalPopup() {
+    const { isShowPopup, selectedTrack, mouseX, mouseY } = this.state;
+    if (isShowPopup) {
+      const style = {
+        position: 'absolute',
+        left: `${mouseX - 100}px`,
+        top: `${mouseY - 200}px`,
+      };
+      return (
+        <div
+          className="popup-container"
+          style={style}
+        >
+          <TrackItemPopupComponent
+            track={selectedTrack}
+          />
+        </div>
+      );
+    }
+    return null;
+  }
+
   render() {
     const { trackList, collectionId } = this.props;
-    const { themeList } = this.state;
-    const { mapInputValue } = this.state;
+    const { themeList, mapInputValue } = this.state;
+    const body = document.querySelector('body');
```

Comment:
[Suggestion]
We don't need to compute to get body every time when it renders.
How about simply using `document.body`?

https://react.dev/reference/react-dom/createPortal#rendering-a-modal-dialog-with-a-portal